addons/website_sale_delivery/static/src/js/website_sale_delivery.js

```diff
@@ -100,6 +100,11 @@ publicWidget.registry.websiteSaleDelivery = publicWidget.Widget.extend({
         if (result.new_amount_total_raw !== undefined) {
             this._updateShippingCost(result.new_amount_total_raw);
         }
+        if (result.new_amount_total_raw !== 0) {
+            document.querySelector('#website_sale_non_free_cart').classList.remove('d-none');
```

Comment:
I think this will raise an error if we don't do -u of website_sale, but the fix should work even if the module has not been updated